src/core/lib/draw-layers.js

```diff
@@ -238,11 +238,12 @@ function drawLayerInViewport({gl, layer, layerIndex, drawPickingColors, glViewpo
 
   if (drawPickingColors) {
     // TODO - Disable during picking
-    Object.assign(moduleParameters, getPickingModuleParameters(layer));
 
     Object.assign(layerParameters, {
       blendColor: [0, 0, 0, (layerIndex + 1) / 255]
     });
+  } else {
+    Object.assign(moduleParameters, getObjectHighlightParameters(layer));
   }
 
   withParameters(gl, layerParameters, () => {
```

Comment:
`layer.drawLayer` already calls `withParameters(this.context.gl, parameters, ...)`. Is it necessary to nest another level here?

Reply:
Good find, fixed.